my_config.yaml

```diff
@@ -0,0 +1,24 @@
+#
```

Comment:
Should this file be a test resource?

Reply:
you mean to move it like this https://github.com/apache/incubator-xtable/blob/main/xtable-utilities/src/test/java/org/apache/xtable/utilities/TestRunCatalogSync.java#L36?